torchchat/distributed/checkpoint.py

```diff
@@ -96,6 +96,7 @@ def _load_checkpoints_from_storage(
         checkpoint_path,
         map_location=builder_args.device,
         mmap=True,
+        weight_only=False,
```

Comment:
Why does it needs false? All LLMs should be loadable with weights_only, shouldn't they? (Also, there are no such option as `weight_only` (or so I hope :P ))
```suggestion
        weights_only=True,
```

Reply:
Good catch on the typo; 

As for setting it to False: I'd rather keep it behavior consistent in a pin bump PR; we can flip in a separate PR